sdk/python/kfp/components_tests/test_components.py

```diff
@@ -741,19 +741,17 @@ def test_input_output_uri_resolving(self):
         task = op(in1='foo')
         resolved_cmd = _resolve_command_line_and_paths(
             component_spec=task.component_ref.spec,
-            arguments=task.arguments,
-            input_uri_generator=lambda name: f"{{{{inputs[{name}].uri}}}}",
-            output_uri_generator=lambda name: f"{{{{outputs[{name}].uri}}}}",
+            arguments=task.arguments
         )
 
         self.assertEqual(
             resolved_cmd.command,
             [
                 'program',
                 '--in1-uri',
-                '{{inputs[In1].uri}}',
+                '{{kfp.output_dir}}/{{workflow.uid}}/{{inputs.parameters.In1-producer-pod-id-}}/In1',
```

Comment:
Are you sure this test works under WIndows?

Reply:
Oh good catch. You're right this won't work.

Will fix this shortly